psydac/polar/dense.py

```diff
@@ -318,11 +318,11 @@ def dot( self, v, out=None ):
         return out
 
     # ...
-    def toarray( self ):
+    def toarray( self, *, order='C' ):
         return self._data.copy()
 
     # ...
-    def tosparse( self ):
+    def tosparse( self, *, order='C' ):
```

Comment:
The `order` argument is not used, plus I cannot see its meaning for a COO matrix

Reply:
it's been deleted